src/user-agent.js

```diff
@@ -1,9 +1,9 @@
 import { arch, platform } from 'os'
+import getPackage from 'load-module-pkg'
 
 import modulePackage from '../package.json'
-import sdkPackage from '../node_modules/sphere-node-sdk/package.json'
 
-const sdkInfo = `node-sdk/${sdkPackage.version}`
+const sdkInfo = `node-sdk/${getPackage('sphere-node-sdk').version}`
```

Comment:
Refactored this to the sphere-node-utils [here](https://github.com/sphereio/sphere-node-utils/pull/28)

Can we update this then, and see if it works in impex